supervisord/datadog_checks/supervisord/supervisord.py

```diff
@@ -163,15 +163,15 @@ def check(self, instance):
     @staticmethod
     def _connect(instance):
         sock = instance.get('socket')
+        user = instance.get('user')
+        password = instance.get('pass')
         if sock is not None:
             host = instance.get('host', DEFAULT_SOCKET_IP)
-            transport = supervisor.xmlrpc.SupervisorTransport(None, None, sock)
+            transport = supervisor.xmlrpc.SupervisorTransport(user, password, sock)
```

Comment:
Could you provide a test case for this ?

Might need to update https://github.com/DataDog/docker-library/blob/master/supervisord/3.3.3

Reply:
Looks good otherwise.